### ISSUE TITLE
[Bug] wifi restore doesn't work on android 14 lineageos 21

### ISSUE THREAD

**Issue by videoman614** [NeoApplications/Neo-Backup#860]
**Description**
Cannot restore wifi on android 14 lineageos 21

**Steps To Reproduce**

1. make backup on old phone android 13
2. copy files after installing android 14 lineageos 21
3. install neo backup, provide permissions 
4. restore wifi
5. reboot
6. wifi network and passwords are not restored

**Expected behavior**
wifi network and password are restored

**Screenshots**
Only if there's no clear logs, add screenshots to help explain your problem.

**System Information(please complete the following information):**

- Device: pixel 7 panther
- Android Version:  14
- ROM: aosp lineageos 21
- App's Version: 8.3.6

-----------------------------

**Additional Notes**

* If you're on the latest Test version(from the TG-Channel or Matrix-Room) please report there directly, not here.

* If you're on the latest github version:
  
  * Please read the [FAQ](https://github.com/NeoApplications/Neo-Backup/blob/master/FAQ.md) first.
  * Please read all the open Issues. If one does match yours, leave a comment there if there's some extra information you can provide.

* If you're not on the latest github version. Update, then make sure that the bug is still there. If it's go further with reporting.

Thanks for reporting.

**Comment by Rorschach1010** [comment]
Same here on CRDroid 10.4. Did did work before an earlier CRDroid A14 versions.

Actually it's the same as with https://github.com/NeoApplications/Neo-Backup/issues/858

**Comment by tatl-im** [comment]
> Same here on CRDroid 10.4. Did did work before an earlier CRDroid A14 versions.
> 
> Actually it's the same as with #858

Hey, that's my bug :D



I've experienced this bug as well as the SMS one, but I want to say that the WiFi AP data does get restored SOMEWHERE.

Here's what I did:
1. Backup old phone with lots of WiFi APs.
2. Check the backup; all of the APs are in the XML file.
3. Restore the backup onto new phone. Even after a reboot, the Saved WiFi APs list in Settings is empty.
4. Make another backup of the NEW phone with the "empty" WiFi AP list.
5. Check the XML file in the NEW backup, all of the APs are there.

So Neo did indeed restore the APs somewhere, which it then took a backup of. But clearly it's not the right place.
